grails-app/controllers/org/pih/warehouse/reporting/ReportController.groovy

```diff
@@ -579,7 +587,7 @@ class ReportController {
                             sw.append('"' + (entry.key?.productCode ?: "").toString()?.replace('"', '""') + '"').append(",")
                             sw.append('"' + (entry.key?.displayNameWithLocaleCode ?: "").toString()?.replace('"', '""') + '"').append(",")
                             sw.append('"' + (entry.key?.productFamily?.name ?: "").toString()?.replace('"', '""') + '"').append(",")
-                            sw.append('"' + (entry.key?.category?.name ?: "").toString()?.replace('"', '""') + '"').append(",")
+                            sw.append('"' + (entry.key?.category?.getHierarchyAsString(" > ") ?: "").toString()?.replace('"', '""') + '"').append(",")
```

Comment:
Was this requested? This could get expensive for lot of rows and will likely would add a lot of noise to the CSV.

Reply:
Yes, it was written in ticket in this way: ```When I open the downloaded report, Then between “Family” and “Category” columns, I can see a column with “Parent Category” OR category appears as [Parent]>[Sub]>[Sub 2]. This option might be easier because there are sometimes several levels of category.```

Reply:
Ok, this could end up causing some performance issues but we'll leave it in for now.